src/glint.gleam

```diff
@@ -840,12 +853,19 @@ fn command_help_to_usage_string(help: CommandHelp, config: Config) -> String {
 fn flags_help_to_string(help: List(FlagHelp), config: Config) -> String {
   use <- bool.guard(help == [], "")
 
+  let longest_flag_length =
+    help
+    |> list.map(flag_help_to_string)
+    |> utils.max_string_length
+    |> int.max(min_column_width)
+
   case config.pretty_help {
     None -> flags_heading
     Some(pretty) -> heading_style(flags_heading, pretty.flags)
   }
   <> {
-    [help_flag_message, ..list.map(help, flag_help_to_string_with_description)]
+    [help_flag, ..help]
+    |> list.map(flag_help_to_string_with_description(_, longest_flag_length + 2))
```

Comment:
❓ Where does this 2 come from?

Reply:
It's actually not from what you quoted there, but it's been added as a configurable `column_gap` value.